### PR TITLE
CLI Test Center Beta Release - 0.1.0

### PR DESCRIPTION
**Pull request by rakayada**

### CODE REVIEW

**Review by robertolopezlopez — CHANGES_REQUESTED** [review]
Hi @rakayada ,

Thanks for this PR. Some fields are not required (for example, bin). Please refactor your changes taking cli-edgeworkers block as example (another node.js package, present in this same file).

Regards,

Roberto

**Comment by rakayada** [comment]
Hi @robertolopezlopez, 

Updated file with suggested change. 


Please do not merge this PR. Request is to just approve. We will merge this PR. 

Thanks!

File: assets/package-list.json [hunk 1]
```diff
@@ -385,7 +385,15 @@
...
 
-
+            {
+                "title": "Test Center",
+                "name": "test-center",
+                "version": "0.1.0",
+                "url": "https://github.com/akamai/cli-test-center",
+                "issues": "https://github.com/akamai/cli-test-center/issues",
+                "commands": [{"name":"test-center","version":"0.1.0","description":"Test Center is a testing tool that checks the effect of configuration changes on your web property. Use this tool as part of your testing protocol to increase your confidence in the safety and accuracy of your configuration changes."}],
+                "requirements": {"go":"1.15"}
```
Comment (robertolopezlopez) [inline assets/package-list.json:395]: I guess that, being a node.js CLI package, this is not really correct

**Review by robertolopezlopez — CHANGES_REQUESTED** [review]
Please, fix the requirements as well.

**Comment by robertolopezlopez** [comment]
> Hi @robertolopezlopez,
> 
> Updated file with suggested change.
> 
> Please do not merge this PR. Request is to just approve. We will merge this PR.
> 
> Thanks!

Hi @rakayada ,

Thank you for your PR but we are in charge of this repository. Thus we are in charge of merging :-)

Kind regards,

Roberto

File: assets/package-list.json (hunk 1 above)
Comment (robertolopezlopez) [reply assets/package-list.json:395]: You also need to change your cli.json or your CLI package will be unusable via akamai CLI https://github.com/akamai/cli-test-center/blob/master/cli.json

Comment (rakayada) [reply assets/package-list.json:395]: We are pushing go CLI Package.

Comment (robertolopezlopez) [reply assets/package-list.json:395]: Understood @rakayada, my apologies. As I told you directly some minutes ago, this was just a misunderstanding from my side.

**Pull request merged**